website/pages/home/content.json

```diff
@@ -0,0 +1,56 @@
+{
+  "principles": [
```

Comment:
I've taken a similar approach here for most pages, moving content into `content.json` files, and replacing `text-and-content` components with the updated `text-splits` component.